### ISSUE TITLE
catalysis scaling relations

### ISSUE THREAD

**Issue by computron** [materialsintelligence/propnet#108]
Should we put catalysis scaling relations in propnet?

This is truly a question (not a request in the form of the question).

On one hand, the catalysis community would likely find it very useful.

On the other hand, these will be complicated to implement (I think?) and it will be something of a minor sub-project to dig these up and get them in there.

**Comment by montoyjh** [comment]
I had the same thought a couple of weeks ago.  I don't know that it'd be advantageous to put every single one, but I think it'd be cool to maybe put some of the more well-known ones like universal BEP scaling and maybe some of the oxygen/carbus species on metals.

**Comment by computron** [comment]
When I talked to Citrine on Wed (July 3) they were already doing this, using graphs of scaling relations to build up ML models

**Comment by montoyjh** [comment]
Yeah, I strongly suspect we'd be behind on this on multiple fronts, and catalysis generally has a lot of inherent challenges related to the fact that data tends to be less material-centric (e. g. is surface or molecule focused), so I won't be making it a priority, but if I have some spare cycles I'll revisit.
